util/template/replace.go

```diff
@@ -0,0 +1,9 @@
+package template
+
+func Replace(s string, replaceMap map[string]string, allowUnresolved bool) (string, error) {
+	t, err := New(s)
```

Comment:
Same here... very unclear what `New` is/does if it doesn't live near its definition